### ISSUE TITLE
Allow user to set the default note-input velocity

### ISSUE THREAD

**Issue by rgreen5** [musescore/MuseScore#22768]
### Your idea

Would it be useful to allow the user an option in Preferences, to choose the default note-input velocity?. Currently this is 64, but AFAIK this is lower than the default for MS3. For some users this may be on the low side?

### Problem to be solved

ww

**Comment by scorster** [comment]
I agree that a value of 64 seems oddly low for the default User velocity in note entry.
